Data/TOR/Story/XML/10588.xml

```diff
@@ -727,74 +727,83 @@ now managed by the millitary.</EnglishText>
       <JapaneseText>一見、平穏な感じを
 受けるかもしれないが、
 この街は非常に危険な状況にある。</JapaneseText>
-      <EnglishText/>
+      <EnglishText>While peaceful on the surface,
+this place is currently in a state of emergency.</EnglishText>
       <Notes/>
       <SpeakerId>15</SpeakerId>
       <Id>49</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2751</PointerOffset>
       <JapaneseText>用がないのなら
 退去するのが望ましい。</JapaneseText>
-      <EnglishText/>
+      <EnglishText>If you have no business here
+I suggest you leave.</EnglishText>
       <Notes/>
       <SpeakerId>15</SpeakerId>
       <Id>50</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2801</PointerOffset>
       <JapaneseText>王朝に関する、様々な
 ウワサが飛び交っているが、
 どれもガセネタばかりだ。</JapaneseText>
-      <EnglishText/>
-      <Notes/>
+      <EnglishText>There are many rumors
+being spread about the
+dynasty, but none are true.</EnglishText>
+      <Notes>王朝 = dynasty. Don't know if this is accurate in context.</Notes>
       <SpeakerId>15</SpeakerId>
       <Id>51</Id>
-      <Status>To Do</Status>
+      <Status>Editing</Status>
     </Entry>
     <Entry>
       <PointerOffset>2823</PointerOffset>
       <JapaneseText>陛下はご存命であられるし、
 ジルバ様が種族主義をあおるような
 演説をしたというのもウソだ。</JapaneseText>
```

Comment:
He's commenting that Her Majesty Agarte is still alive, and that  Zilva totally did not put on a racist speech nosiree. (basically nothing to see here).